src/main/kotlin/com/toasttab/pulseman/view/UserFeedbackUI.kt

```diff
@@ -15,58 +15,71 @@
 
 package com.toasttab.pulseman.view
 
+import androidx.compose.foundation.ScrollState
 import androidx.compose.foundation.background
 import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Column
 import androidx.compose.foundation.layout.Row
 import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
 import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.text.selection.SelectionContainer
+import androidx.compose.foundation.verticalScroll
 import androidx.compose.material.Icon
 import androidx.compose.material.IconButton
-import androidx.compose.material.TextField
+import androidx.compose.material.Text
 import androidx.compose.material.icons.Icons
 import androidx.compose.material.icons.filled.Delete
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.text.TextRange
-import androidx.compose.ui.text.input.TextFieldValue
+import androidx.compose.ui.text.style.TextOverflow
 import androidx.compose.ui.unit.dp
 import com.toasttab.pulseman.AppStrings.CLEAR_LOG
 import com.toasttab.pulseman.AppTheme
 
 /**
  * This views shows a scrollable history of events in the current tab.
  * It is used by passing the setUserFeedback function to all views that need it.
- *
- * TODO get selectable text working in this view
  */
 @Composable
 fun userFeedbackUI(
-    userFeedback: String,
+    userFeedback: List<String>,
     onUserFeedbackClear: () -> Unit
 ) {
-    Row {
-        TextField(
-            label = { },
-            value = TextFieldValue(
-                text = userFeedback,
-                // Force the userfeedback to always show the end of the text
-                selection = TextRange(userFeedback.length)
-            ),
-            readOnly = true,
-            onValueChange = {},
-            trailingIcon = {
-                IconButton(
-                    onClick = { onUserFeedbackClear() }
-                ) {
-                    Icon(Icons.Default.Delete, contentDescription = CLEAR_LOG)
-                }
-            },
+    SelectionContainer {
+        Row(
             modifier = Modifier
-                .padding(1.dp)
                 .fillMaxSize()
                 .background(color = AppTheme.colors.backgroundLight)
-                .border(2.dp, AppTheme.colors.backgroundMedium)
-                .padding(2.dp)
-        )
+                .border(width = 2.dp, color = AppTheme.colors.backgroundMedium)
+                .padding(all = 3.dp),
+        ) {
+            Column(
+                modifier = Modifier
+                    .verticalScroll(state = ScrollState(0), reverseScrolling = true)
```

Comment:
We revert the scrolling direction and force it to always be at index 0, so it's actually going to be scrolled all the way all the time, so when we add a message, the column will scroll and we will be able to see it.